dashboard/app/views/teacher_mailer/hour_of_code_tutorial_email.html.haml

```diff
@@ -0,0 +1,37 @@
+%p
+  = t('hoc_tutorial_email_content.dear_teacher', teacher_name: @teacher_name)
```

Comment:
Not 100% sure on how they work differently, but I needed the `I18n.t` to get the passed in urls to correctly hyperlink text (rather than printing out the entire url)